src/CoreBundle/Bridge/Chameleon/Module/Sidebar/SidebarBackendModule.php

```diff
@@ -0,0 +1,257 @@
+<?php
+
+/*
+ * This file is part of the Chameleon System (https://www.chameleonsystem.com).
+ *
+ * (c) ESONO AG (https://www.esono.de)
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace ChameleonSystem\CoreBundle\Bridge\Chameleon\Module\Sidebar;
+
+use ChameleonSystem\CoreBundle\Response\ResponseVariableReplacerInterface;
+use ChameleonSystem\CoreBundle\Service\LanguageServiceInterface;
+use ChameleonSystem\CoreBundle\Util\InputFilterUtilInterface;
+use ChameleonSystem\CoreBundle\Util\UrlUtil;
+use Symfony\Component\HttpFoundation\RequestStack;
+
+class SidebarBackendModule extends \MTPkgViewRendererAbstractModuleMapper
+{
+    private const DISPLAY_STATE_SESSION_KEY = 'chameleonSidebarBackendModuleDisplayState';
+
+    /**
+     * @var LanguageServiceInterface
+     */
+    private $languageService;
+    /**
+     * @var UrlUtil
+     */
+    private $urlUtil;
+    /**
+     * @var RequestStack
+     */
+    private $requestStack;
+    /**
+     * @var InputFilterUtilInterface
+     */
+    private $inputFilterUtil;
+    /**
+     * @var ResponseVariableReplacerInterface
+     */
+    private $responseVariableReplacer;
+
+    public function __construct(LanguageServiceInterface $languageService, UrlUtil $urlUtil, RequestStack $requestStack, InputFilterUtilInterface $inputFilterUtil, ResponseVariableReplacerInterface $responseVariableReplacer)
+    {
+        parent::__construct();
+        $this->languageService = $languageService;
+        $this->urlUtil = $urlUtil;
+        $this->requestStack = $requestStack;
+        $this->inputFilterUtil = $inputFilterUtil;
+        $this->responseVariableReplacer = $responseVariableReplacer;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function Accept(\IMapperVisitorRestricted $oVisitor, $bCachingEnabled, \IMapperCacheTriggerRestricted $oCacheTriggerManager)
+    {
+        $this->restoreDisplayState();
+        $oVisitor->SetMappedValue('sidebarToggleNotificationUrl', $this->getSidebarToggleNotificationUrl());
+        $oVisitor->SetMappedValue('menuItems', $this->getMenuItems());
+    }
+
+    private function restoreDisplayState(): void
+    {
+        $session = $this->requestStack->getCurrentRequest()->getSession();
+        $displayState = $session->get(self::DISPLAY_STATE_SESSION_KEY, '');
+        if ('minimized' === $displayState) {
+            $value = 'sidebar-minimized';
```

Comment:
Is this a css class?
I would use "minimized" then. The CSS would need to be specific anyway. Something like .sidebar & minimized